data/hai/unfettered jobs.txt

```diff
@@ -0,0 +1,458 @@
+				decline
+
+mission "Unfettered Challenge[1]"
+	name `Defeat the captain`
+	description `An Unfettered captain has provoked you in a duel. Disable his ship for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		has "Unfettered: Prove Your Strength"
+		random < 40
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		personality disables staying heroic target
+		fleet
+			names "hai"
+			variant
+				"Sea Scorpion (Pulse)" 2
+		conversation
+			action
+				payment 400000
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 30
+				fail "Unfettered Challenge[1]"
+			`You receive a message from the <npc> : "Raaaah... Well you have beaten me alright, so I'll pay up... Here are the <payment> credits, captain. Next time I will not underestimate you anymore!"`
+				decline
+
+mission "Unfettered Challenge[2]"
+	name `Defeat the experienced captain`
+	description `An experienced Unfettered captain has provoked you into a duel. Disable his ship for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	to offer
+		has "Unfettered Jump Drive 3: offered"
+		has "Unfettered: Prove Your Strength"
+		random < 10
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		personality disables staying heroic target
+		fleet
+			names "hai"
+			variant
+				"Shield Beetle (Pulse)"
+				"Sea Scorpion (Pulse)"
+		dialog `You seem to learn from our encounters as well... Well next time it will be different.`
+		conversation
+			action
+				payment 600000
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 60
+				fail "Unfettered Challenge[2]"
+			`You receive a message from the <npc> : "Nicely done! You seem to learn from our encounters as well... But next time will be different, I assure you. For now, here are the agreed <payment> credits, captain. Hopefully it allows you to improve the <ship> before our next battle..."`
+				decline
+
+
+
+mission "Get Jump Drives(1)"
+	name `Collect a jump drive`
+	description `Go to the pirate world of <stopovers> and recover a jump drive from a vault, then bring to <destination> for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	destination "Darkcloak"
+	to offer
+		random < 30
+		has "Unfettered: Prove your strength 2: done"
+	stopover
+		distance 12 18
+		government "Pirate"
+	on stopover
+		dialog `You manage to land and retrieve the jump drive without too much trouble. Do not rely on luck too long though, you had better depart quickly!`
+		outfit "Jump Drive"
+	on visit
+		dialog `You have a mission to bring a jump drive here. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`
+	on complete
+		payment 1000000
```

Comment:
Its already quite a long way to get there, and these missions are quite uncommon, I placed them at this price to make you sometimes want to take them given how much money tribute gives, or there is no point in this mission from a gameplay perspective
They really want their JD, too, but maybe I could make it be 800 000?

Reply:
Maybe, I was thinking a bit less as you're traveling through mostly peaceful space and you could potentially be friendly with pirate which left just distance being the factor but player being friendly with all the factions at once is probably unlikely enough.

Reply:
You strike a hard bargain

For context, you get payed 1M for going into friendly Hai space taking tributes of 50 tons, easily.
This may take less cargo, but it will be into probable enemy territory, where you may have to pay a bribe only give 500k?

Reply:
50/50 for delivering it but not supplying it sounds about right.

At this point for the players, pirates should be an inconvenience more than a dire threat, as well.

That's a large enough distance, too, that smart players will make additional money moving commodities back and forth anyway.